internal/formatters/util.go

```diff
@@ -73,9 +89,10 @@ type UserResponse struct {
 
 // resultsText represents the results of check execution against the asset.
 type resultsText struct {
-	Passed []checkExecutionInfo `json:"passed" xml:"passed"`
-	Failed []checkExecutionInfo `json:"failed" xml:"failed"`
-	Errors []checkExecutionInfo `json:"errors" xml:"errors"`
+	Passed  []checkExecutionInfo `json:"passed" xml:"passed"`
+	Failed  []checkExecutionInfo `json:"failed" xml:"failed"`
+	Errors  []checkExecutionInfo `json:"errors" xml:"errors"`
+	Warning []checkExecutionInfo `json:"warning,omitempty" xml:"warning,omitempty"`
```

Comment:
```suggestion
	Warnings []checkExecutionInfo `json:"warning,omitempty" xml:"warning,omitempty"`
```

Reply:
Another note: I'm glad we're omitting this if it's empty, but we should make sure the backend doesn't expect it if we don't send it. (E.g. make sure it can be omitted there too). No action required here, just something to note to you since you're working with implementation on the backend

Reply:
This is optional in the [API spec](https://catalog.redhat.com/api/containers/v1/ui/#/Certification%20projects/graphql.cert_project_test_result.create_certification_test_result).